src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/SeachUserListRow.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import { screen } from '@testing-library/react';
+import { render, wrapInIntlProvider } from 'lib-tests';
+
+import { SearchUserListRow } from './SeachUserListRow';
+
+describe('SearchUserListRow', () => {
+  it('should render name and email', () => {
+    render(
+      wrapInIntlProvider(
```

Comment:
Don't need `wrapInIntlProvider`, `render` use it already.
Same for the other tests.